examples/dashboard-streamlit/README.md

```diff
@@ -0,0 +1,116 @@
+        )
+        y = np.unique(y, return_inverse=True)[1]
+
+    else:
+        X = None
+        y = None
+
+    return X, y
+
+
+@st.cache
+def create_figure(X, y):
+    umap_3d = UMAP(n_components=3, init="random", random_state=0)
+    proj_3d = umap_3d.fit_transform(X, y=y)
+
+    fig = px.scatter_3d(proj_3d, x=0, y=1, z=2, color=y)
+    fig.update_layout(transition_duration=500, height=1000)
+    fig.update(layout_coloraxis_showscale=False)
+    fig.update_traces(marker_size=2)
+
+    return fig
+```
+
+
+### Define the App Layout and Content
+
+After creating the functions, define the layout of the Streamlit app. Streamlit makes it easy to define text content with functions like `title`, `markdown`, and `text`. You can also define user input using functions like `selectbox` or output using functions like `plotly_chart`.
+
+The following code displays a title with a markdown text description and creates two columns. The first column contains a dropdown selector for the dataset, and the second a chart showing the 3D graph of the UMAP projection.
+
+See the [Streamlit documentation](https://docs.streamlit.io/) for more information about the functions used here and more.
+
+``` python
+st.set_page_config(page_title="UMAP Projections", layout="wide")
+
+st.title("UMAP Projections for MNIST and Fashion-MNIST Datasets")
+
+st.markdown(
+    """
+    Uniform Manifold Approximation and Projection (UMAP) is a general-purpose dimension reduction algorithm. Similar to t-distributed stochastic neighbor embedding (t-SNE), you can use UMAP to visualize the relationships between data points. In this example, we are training a three-component UMAP model on MNIST datasets and then displaying the 3D graph of the result. The color of the point in the graph is based on the label. In the resulting graph, blobs of colors show that UMAP clustered data points with similar labels together.
+    """
+)
+
+col1, col2 = st.columns([1, 3])
+
+with col1:
+    selected_dataset = st.selectbox("Please select a dataset:", ("MNIST-Digits", "MNIST-Fashion"))
+
+    data_load_state = st.text("Loading data...")
+    X, y = load_data(selected_dataset)
+    data_load_state.text("Done loading data!")
+
+with col2:
+    fig = create_figure(X, y)
+    plot_3d = st.plotly_chart(create_figure(X, y), use_container_width=True)
+```
+
+
+### Run the App
+
+You then need to only load the app code to git and link the code appropriately to a Saturn Cloud deployment. 
+
+[Click here]("https://app.community.saturncloud.org/dash/resources?recipeUrl=https://raw.githubusercontent.com/saturncloud/examples/main/examples/dashboard-dash/.saturn/saturn.json") to see how this is accomplished. 
+
+## Conclusion
+Streamlit makes it easy to develop interactive apps, and Saturn Cloud makes it easy to deploy these apps. Check out our other [dashboard resources](https://saturncloud.io/docs/examples/python/production/) for examples using other frameworks.
```

Comment:
I think this can be part of the previous section

```suggestion
Check out our other [Python dashboard resources](https://saturncloud.io/docs/examples/python/production/) for examples using other frameworks.
```